lib/github_changelog_generator/generator/generator_tags.rb

```diff
@@ -69,6 +95,7 @@ def version_of_first_item
     # @return [Array]
     def get_filtered_tags(all_tags)
       filtered_tags = filter_since_tag(all_tags)
+      filtered_tags = filter_due_tag(filtered_tags)
```

Comment:
Noticed `filter_due_tag` was not called previously. I think that is a bug, so I added it.